aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie/Utilities/FileHelpers.cs

```diff
@@ -57,6 +57,10 @@ public class FileHelpers
             {
                 modelState.AddModelError(formFile.Name, $"The {fieldDisplayName}file ({fileName}) is empty.");
             }
+            else if (formFile.Length > 1048576)
+            {
+                modelState.AddModelError(formFile.Name, $"The {fieldDisplayName}file ({fileName}) exceeds 1 MB.");
```

Comment:
Should there be a space between `{fieldDisplayName}` and `file`?

Reply:
The space is added as part of the `fieldDisplayName`; so if the field is empty, an extra space isn't present in the string.